### PR TITLE
Removing orchestrator internal IDP setup

### PR DESCRIPTION
**Pull request by mohitseth-nitrr**

### CODE REVIEW

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: orchestrator/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## orchestrator
+
+This directory stores the script to setup aifabric connection related properties in orchestartor. This script has to be executed throuh powershell in Administrator mode & before running script set execution policy to RemoteSigned by running "Set-ExecutionPolicy RemoteSigned".
+
+For AI Center < 21.4, use orchestratorAutomation.ps1
+For AI Center >= 21.4, use orchestratorAutomationLatest.ps1
```
Comment (vivekjhauipath) [inline orchestrator/README.md:6]: nit: use _21.4 or something instead of latest

Comment (mohitseth-nitrr) [reply orchestrator/README.md:6]: Actually going forward only this should be used ... but i will confirm naming with jeremy

**Review by Mdirfan6 — APPROVED** [review]
lgtm

**Pull request merged**